deploy/templates/nstemplatetiers/intellarge/ns_dev.yaml

```diff
@@ -252,6 +252,19 @@ objects:
                 kubernetes.io/metadata.name: redhat-ods-applications
     policyTypes:
       - Ingress
+- apiVersion: networking.k8s.io/v1
```

Comment:
BTW, we need to delete all intel tiers :) But let's do it in a separate PR.

Reply:
got it, I'll open a new one, once those are merged!

Reply:
should we keep them , just in case they might be used by the intel team for their own kubesaw instance ?

Reply:
We can always revert the cleanup PRs later if needed. But most likely we won't need these tiers anytime soon.